spec.html

```diff
@@ -37,6 +37,151 @@ <h1>Introduction</h1>
+          1. Let O be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateError.prototype%"`, « [[ErrorData]] »).
+          1. Let _errorsIterator_ be ? GetMethod(_errors_, @@iterator).
+          1. Let _errorsList_ ? IterableToList(_errors_, _errorsIterator_).
+          1. Let _errorsArray_ be CreateArrayFromList(_errorsList_).
+          1. Perform ! CreateDataProperty(_O_, `"errors"`, _errorsArray_).
+          1. If _message_ is not _undefined_, then
+          1. Let msg be ? ToString(_message_).
+          1. Let msgDesc be the PropertyDescriptor { [[Value]]: _msg_, [[Writable]]: *true*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.
+          1. Perform ! DefinePropertyOrThrow(_O_, `"message"`, _msgDesc_).
+          1. Return _O_.
+        </emu-alg>
+      </emu-clause>
+    </emu-clause>
+
+    <emu-clause id="sec-properties-of-the-aggregateerror-constructors">
+      <h1>Properties of the _AggregateError_ Constructor</h1>
+      <p>The _AggregateError_ constructor:</p>
+      <ul>
+        <li>has a [[Prototype]] internal slot whose value is the intrinsic object %Error%.</li>
+        <li>has a `name` property whose value is the String value `"<var>AggregateError</var>"`.</li>
+        <li>has the following properties:</li>
+      </ul>
+
+      <emu-clause id="sec-aggregate-error.prototype">
+        <h1>AggregateError.prototype</h1>
+        <p>The initial value of <code><var>AggregateError</var>.prototype</code> is a _AggregateError_ prototype object (<emu-xref href="#sec-properties-of-the-aggregateerror-prototype-objects"></emu-xref>).</p>
+        <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
+      </emu-clause>
+    </emu-clause>
+
+    <emu-clause id="sec-properties-of-the-aggregateerror-prototype-objects">
+      <h1>Properties of the _AggregateError_ Prototype Object</h1>
+      <p>The _AggregateError_ prototype object:</p>
+      <ul>
+        <li>is an ordinary object.</li>
+        <li>is not an Error instance and does not have an [[ErrorData]] internal slot.</li>
+        <li>has a [[Prototype]] internal slot whose value is the intrinsic object %Error.prototype%.</li>
+      </ul>
+
+      <emu-clause id="sec-aggregate-error.prototype.constructor">
+        <h1>_AggregateError_.prototype.constructor</h1>
+        <p>The initial value of the `AggregateError.prototype.constructor` is the intrinsic object %_AggregateError_% (<emu-xref href="#sec-aggregate-error-constructors"></emu-xref>).</p>
+      </emu-clause>
+
+      <emu-clause id="sec-aggregate-error.prototype.message">
+        <h1>_AggregateError_.prototype.message</h1>
+        <p>The initial value of the `AggregateError.prototypemessage` is the empty String.</p>
+      </emu-clause>
+
+      <emu-clause id="sec-aggregate-error.prototype.name">
+        <h1>_AggregateError_.prototype.name</h1>
```

Comment:
also this one?

Reply:
`sec-error.prototype.name` should be updated in that case. I'll do it

Reply:
hm, no, this should be "The initial value of the `AggregateError.prototypename` is the String value `AggregateError`"